src/Symfony/Component/Intl/composer.json

```diff
@@ -24,13 +24,16 @@
         }
     ],
     "require": {
-        "php": ">=8.2"
+        "php": ">=8.2",
+        "symfony/deprecation-contracts": "^2.5|^3"
     },
     "require-dev": {
         "symfony/filesystem": "^6.4|^7.0",
-        "symfony/finder": "^6.4|^7.0",
         "symfony/var-exporter": "^6.4|^7.0"
     },
+    "conflict": {
+        "symfony/string": "<7.1"
```

Comment:
Why do we need the conflict?

Reply:
Better link to this message than trying to rephrase it (i'd fail)
https://github.com/symfony/symfony/pull/53096#pullrequestreview-1800071133